### ISSUE TITLE
Unable to build cache error not showing the right file in the file explorer.

### ISSUE THREAD

**Issue by muchitto** [vshaxe/vshaxe#488]
It is pretty annoying that when an error occurs (maybe you have closed the visual studio code and later done a git pull and opened visual studio code, it just reports that "Unable to build cache - completion features may be slower than expected. Try fixing the error(s) and restarting the language server." and does not highlight the erroneous file in the file explorer as red until you actually open the file for editing. I have to always look at the compilation server output to see where the problem is (or just open every file until it highlights the correct file with the errors), fix it and then manually restart the Haxe language server (also a pretty tedious task, can't this be automated if there was an error compiling and changes had been made to an erroneous file?). Is there something that can be done to better this workflow? :)

**Comment by Gama11** [comment]
With some work it should be possible to show cache building errors in the problems panel. Had this idea a while ago already, just haven't implemented it so far.

**Comment by muchitto** [comment]
Yeah, because the amount times I have to restart the language server while programming is kinda ridiculous and most certainly not user friendly.
